google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/AckReplyConsumerWithResponse.java

```diff
@@ -18,8 +18,23 @@
 
 import java.util.concurrent.Future;
 
+/**
```

Comment:
Please mention that the guarantees work with exactly once
Otherwise we will always return okay and messages can be redelivered

Reply:
Updated comment